collections/css-resets/index.md

```diff
@@ -0,0 +1,15 @@
+---
+items:
+ - necolas/normalize.css
+ - jgthms/minireset.css
+ - csstools/sanitize.css
+ - vladocar/CSS-Micro-Reset
+ - zacanger/styled-reset
+ - jeroenoomsNL/initialize-css
+ - BaseMax/ResetCSS
+ - runnez/reboot.css
+
+display_name: CSS Resets
+created_by: krishdevdb
+---
+CSS Resets Are Used My Millions Of Industries To Make Their Website Look The Same On Every Browser
```

Comment:
Small nit: Can we use regular sentence case, instead of title case here? That's what other collections use, from what I can tell. Thanks!

Reply:
Thanks for reviewing @gallexi. I have now updated the case. Anything Else I can Do to make it perfect.